src/themes/greyVest/SearchFilters.js

```diff
@@ -32,7 +45,7 @@ let BasicSearchFilters = ({ setMode, trees, children, FiltersBox }) => (
     </LinkButton>
   </div>
 )
-let BuilderSearchFilters = ({ setMode, trees, QueryBuilder }) => (
+let BuilderSearchFilters = ({ setMode, trees }) => (
```

Comment:
you removed QueryBuilder from the props but left it in the invocation on line 69